proto/qgb/query.proto

```diff
@@ -14,13 +15,40 @@ service Query {
   rpc ValsetConfirm(QueryValsetConfirmRequest) returns (QueryValsetConfirmResponse) {
     option (google.api.http).get = "/qgb/valset/confirm";
   }
+  // ValsetConfirmsByNonce
+  rpc ValsetConfirmsByNonce(QueryValsetConfirmsByNonceRequest) returns (QueryValsetConfirmsByNonceResponse) {
+    option (google.api.http).get = "/qgb/confirms/{nonce}";
+  }
   // GetDelegateKeyByOrchestrator Queries delegate key by orchestrator
   rpc GetDelegateKeyByOrchestrator(QueryGetDelegateKeysByOrchestratorAddress) returns (QueryGetDelegateKeyByOrchestratorResponse) {
     option (google.api.http).get = "/qgb/query_delegate_keys_by_orchestrator";
   }
   // this line is used by starport scaffolding # 2
 }
 
+// FIXME: Should we also have support for `Valset`s ? Or only `ValsetConfirm`s ?
```

Comment:
We might need to add valsets when we implement slashing, but for now we use should only need the confirms.

The only reason that we're having validators submit transactions for the MVP is because it makes aggregation trivial for the relayer